imports/api/units.js

```diff
@@ -353,7 +354,9 @@ let Units
 if (Meteor.isClient) {
   Units = new Mongo.Collection(collectionName)
   Units.helpers({
-    metaData: () => UnitMetaData.findOne({bzId: this.id}),
+    metaData () {
```

Comment:
Good catch! I'm guessing this was changed because otherwise `this.id` would be undefined. The other method (`rolesData`) need to be modified to work the same